### ISSUE TITLE
Property aria-details-id does not work for scale-dropdown-select field

### ISSUE THREAD

**Issue by EdsonT-Systems** [telekom/scale#2443]
**Scale Version**
3.0.0-beta.151

**Framework and version**
Angular latest with auto-generated scale wrapper packages

**Current Behavior**
When using the aria-details-id property on the scale-dropdown-select field and a div with an id to have the additional information read, the information is not being read by the screen reader (NVDA).

**Expected Behavior**
The expected behavior is the one that occurs in the other fields (text-field, text-area, etc.), where the property does work correctly and is read by the screen reader.

**Code Reproduction**

html:

<img width="866" height="496" alt="Image" src="https://github.com/user-attachments/assets/bcad6b0c-b78d-41dd-b7bf-a8a66f371a35" />

ts:

<img width="967" height="647" alt="Image" src="https://github.com/user-attachments/assets/5b68dfdb-eca6-4632-be0e-f72ff60eb2d5" />

ui:

<img width="305" height="127" alt="Image" src="https://github.com/user-attachments/assets/c44ffbb8-0648-4dd4-9799-c1ca549d7e2d" />

**Desktop (please complete the following information):**
 - Browser chrome
 - Version latest

**Additional context**

The exact same steps were reproduced for the other fields and they did work. The only one that seems not to work is this one. It couldn’t be replicated in the sandbox because it runs very slowly.
